src/auth/dto/register.dto.ts

```
@@ -0,0 +1,5 @@
+import { IsEmail, IsString, MinLength } from 'class-validator';
```

ESLint

[error] 'IsEmail' is defined but never used.

(@typescript-eslint/no-unused-vars)